### ISSUE TITLE
Proxy: container dies after proxy update

### ISSUE THREAD

**Issue by raladev** [ExchangeUnion/xud-docker#801]
Description:
Sometimes, after proxy container upgrade (changing from one branch to another with new proxy image), it falls down with the err.

```
[entrypoint] Waiting for xud tls.cert to be created...
[entrypoint] Waiting for xud tls.cert to be created...
time="2020-11-25T14:49:14Z" level=info msg="Parsing command-line arguments"
time="2020-11-25T14:49:14Z" level=info msg="Creating service manager"
time="2020-11-25T14:49:14Z" level=debug msg="[watch] Neutrino syncing"
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x9e9aa0]

goroutine 6 [running]:
github.com/ExchangeUnion/xud-docker-api-poc/service.(*SingleContainerService).GetContainer(0xc000172e10, 0x5, 0xc000064ee0, 0x1)
    /src/backend/service/single_container_service.go:37 +0x40
github.com/ExchangeUnion/xud-docker-api-poc/service/lnd.(*LndService).watchNeutrinoSyncing(0xc00069ea80, 0x0, 0x0)
    /src/backend/service/lnd/lnd.go:253 +0xb3
github.com/ExchangeUnion/xud-docker-api-poc/service/lnd.New.func1(0xc00069ea80)
    /src/backend/service/lnd/lnd.go:91 +0x2f
created by github.com/ExchangeUnion/xud-docker-api-poc/service/lnd.New
    /src/backend/service/lnd/lnd.go:90 +0x3d0
```

**Comment by kilrau** [comment]
This one looks more critical. Did you figure out how to reproduce this?

**Comment by reliveyy** [comment]
> This one looks more critical. Did you figure out how to reproduce this?

I can repro it in development. And it should be fixed in #803.

**Comment by raladev** [comment]
was not able to  reproduce after 803 merge, Closed.
